pkg/collector/corechecks/system/disk/disk/disk_windows.go

```diff
@@ -0,0 +1,95 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+
+package disk
+
+import (
+	"fmt"
+	win "golang.org/x/sys/windows"
+	"unsafe"
+)
+
+func (c *Check) fetchAllDeviceLabelsFromLsblk() error {
+	return nil
+}
+
+func (c *Check) fetchAllDeviceLabelsFromBlkidCache() error {
+	return nil
+}
+
+func (c *Check) fetchAllDeviceLabelsFromBlkid() error {
+	return nil
+}
+
+var mpr = win.NewLazySystemDLL("mpr.dll")
+var procWNetAddConnection2W = mpr.NewProc("WNetAddConnection2W")
+
+type netResource struct {
+	Scope       uint32
+	Type        uint32
+	DisplayType uint32
+	Usage       uint32
+	localName   *uint16
+	remoteName  *uint16
+	comment     *uint16
+	provider    *uint16
+}
+
+// NetAddConnection specifies the command used to add a new network connection.
+var NetAddConnection = func(_mountType, localName, remoteName, password, username string) error {
+	return wNetAddConnection2(localName, remoteName, password, username)
+}
```

Comment:
The Python version seems to have specific handling depending on the type, don't we need to do the same ?

Reply:
You mean what we do in the `RemotePath` method?